src/TransferInformation/BaseTransferInformation.php

```diff
@@ -202,6 +207,16 @@ public function getInstructionId(): ?string
         return $this->instructionId;
     }
 
+    public function setCategoryPurposeCode(string $categoryPurposeCode): void
+    {
+        $this->categoryPurposeCode = $categoryPurposeCode;
+    }
+
+    public function getCategoryPurposeCode(): ?string
```

Comment:
You need to add this as a stub to `TransferInformationInterface` and PHPStan will start passing.